core-plugins/src/main/java/io/cdap/plugin/batch/source/FTPBatchSource.java

```diff
@@ -45,9 +45,11 @@
 
 /**
  * {@link BatchSource} that reads from an FTP or SFTP server.
+ * This plugin is deprecated. Use https://github.com/data-integrations/ftp-plugins instead.
  */
+@Deprecated
 @Plugin(type = "batchsource")
-@Name("FTP")
+@Name("FTPDeprecated")
```

Comment:
If we change the name of the plugin, then it wont be backwards compatible for the case where user has a pipeline with existing plugins in system scope. Upgrading that pipeline will not find plugin with this name. Is my understanding correct?

Reply:
+1. Should only update display name to ``FTP (Deprecated)`` in the widget JSON.

Reply:
Yeah this is not good; I'll revert it.
@ajainarayanan said that widget json spec doesn't allow us to set a custom name. The display name is obtained from the plugin name (camel case name is split by space). @bdmogal how should we proceed?

Reply:
@rmstar 
I think we can use `"display-name": "",` for the plugin widget.

Reply:
https://cdap.atlassian.net/wiki/spaces/DOCS/pages/480379311/Plugin+Presentation

Reply:
done, thanks